### PR TITLE
chore: No restricted imports lint

### PR DESCRIPTION
**Pull request by KiranVarma02**
To save time during code reviews

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -86,6 +86,19 @@ module.exports = {
           }
         ],
 
+        // Check relative import paths are used
+        'no-restricted-imports': [
+          'error',
+          {
+            patterns: [
+              {
+                group: ['../'],
```
Comment (colinrotherham) [inline .eslintrc.cjs:95]: Nice one. What about sibling imports?

```suggestion
                group: ['./', '../'],
```

Comment (KiranVarma02) [reply .eslintrc.cjs:95]: I thought we use `./` in some places? let me check

File: .eslintrc.cjs [hunk 2]
```diff
@@ -86,6 +86,19 @@ module.exports = {
           }
         ],
 
+        // Check relative import paths are used
```
Comment (colinrotherham) [inline .eslintrc.cjs:89]: I think we're making sure we _don't_ use relative imports?

We're trying to stick with `~` aliases instead (so files can be moved without changing their imports)

```suggestion
        // Check relative import paths use aliases
```

File: .eslintrc.cjs (hunk 1 above)
Comment (KiranVarma02) [reply .eslintrc.cjs:95]: in 1 place :), I will correct that and add it to the rule

Comment (KiranVarma02) [reply .eslintrc.cjs]: done

File: .eslintrc.cjs [hunk 3]
```diff
@@ -86,6 +86,19 @@ module.exports = {
...
         ],
 
+        // Check relative import paths use aliases
+        'no-restricted-imports': [
+          'error',
+          {
+            patterns: [
+              {
+                group: ['./', '../'],
+                message: 'Relative imports are not allowed.'
```
Comment (colinrotherham) [inline .eslintrc.cjs:96]: Sorry, I forgot this message also said relative imports not aliases

Unless the default is obvious enough and we don't need one?

Thanks @KiranVarma02

```suggestion
                message: "Please use '~/*' import alias instead.'
```

Comment (KiranVarma02) [reply .eslintrc.cjs:96]: perfect

**Review by colinrotherham — APPROVED** [review]
Thanks @KiranVarma02

Want to keep this open and open PRs across the other repos too?

Can come back to it if we discover any other tweaks are needed

**Pull request merged**